lib/element.ts

```diff
@@ -1,8 +1,10 @@
 let webdriver = require('selenium-webdriver');
+let NoSuchElementError = require('selenium-webdriver/lib/error').NoSuchElementError;
```

Comment:
So, for internal reasons, any time we do `require('selenium-webdriver/lib/<anything>)` our import process for Google gets a tiny bit harder. Can we revert back to just using `webdriver.error.NoSuchElementError`?